srcpkgs/libglvnd/files/LICENSE

```diff
@@ -0,0 +1,26 @@
+Copyright (c) 2013, NVIDIA CORPORATION.
```

Comment:
Pls don't vendor licenses in `void-packages`

Reply:
Isn't that basically a MIT license? [here](https://gitlab.freedesktop.org/glvnd/libglvnd/merge_requests/23) sounds like it should be included. Do we need to include also the whole 'Acknowledgements' section of the `README.md`?

Reply:
so what's the deal with this?

Reply:
We don't vendor the license anymore